client/actions.js

```diff
@@ -0,0 +1,18 @@
+export const GO_HOME = 'GO_HOME'
+export const UPDATE_STATUS = 'UPDATE_STATUS'
+
+export function goHome() {
+  return {
+    type: GO_HOME
```

Comment:
not sure this is going to be used, we should talk to @tony-luisi about how thr router is is going to work